packages/chain-adapters/src/utxo/UtxoBaseAdapter.ts

```diff
@@ -133,14 +133,14 @@ export abstract class UtxoBaseAdapter<T extends UtxoChainId> implements IChainAd
   getBIP44Params({
     accountNumber,
     accountType,
-    index = 0,
+    index,
     isChange = false,
   }: GetBIP44ParamsInput): BIP44Params {
     if (accountNumber < 0) {
       throw new Error('accountNumber must be >= 0')
     }
 
-    if (index < 0) {
+    if (index !== undefined && index < 0) {
```

Comment:
@kaladinlight this was the only logical change in the last commit with updated tests - `0` is falsy but a valid index obv

Reply:
ah yes, better to be explicit. would have still worked coincidentally, but this is the right call